packages/components/src/local/organisms/planning-messages-list/index.tsx

```diff
@@ -125,16 +104,39 @@ export const PlanningMessagesList: React.FC<PropTypes> = ({
       // check if message is being edited
       const localTemplates = templates || []
       const template = localTemplates.find((value: TemplateBody) => value.title === message.details.messageType)
-      setSelectScema(template)
       if (!template) {
         console.log('template not found for', message.details.messageType, 'templates:', templates)
       }
       if (message && template) {
+        const saveMessage = () => {
+          if (messageValue.current) {
+            const details: MessageDetails = {
+              channel: channel.uniqid,
+              from: {
+                force: selectedForce.name,
+                forceColor: selectedForce.color,
+                roleName: selectedRoleName,
+                roleId: playerRoleId,
+                iconURL: selectedForce.iconURL || selectedForce.icon || ''
+              },
+              messageType: template.title,
```

Comment:
@lilitkarapetyan - I'd like use to store the template id in `messageType`, please.  That means Serge won't fail if the template title is changed.